### ISSUE TITLE
Abseil symbolize doesn't work on Windows Bazel RBE

### ISSUE THREAD

**Issue by veblush** [grpc/grpc#24627]
Abseil symbolize doesn't work on Windows (#24294). This happens on Bazel RBE only so it could be caused by RBE setup.

I ran the //test/core/extra test from the PR above on Kokoro Win2016 image and it passed properly.

```
$ bazel run //test/core/extra
Executing tests from //test/core/extra:extra
-----------------------------------------------------------------------------
[==========] Running 2 tests from 1 test suite.
[----------] Global test environment set-up.
[----------] 2 tests from ExtraTest
[ RUN      ] ExtraTest.Basic
get_trace()=[Stack trace:
    @   00007FF6329A1CD7  (unknown)  ExtraTest_Basic_Test::TestBody
    @   00007FF6329D62BC  (unknown)  testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test,void>
    @   00007FF6329D6062  (unknown)  testing::internal::HandleExceptionsInMethodIfSupported<testing::Test,void>
    @   00007FF6329C041A  (unknown)  testing::Test::Run
    @   00007FF6329C0E45  (unknown)  testing::TestInfo::Run
    @   00007FF6329C14E7  (unknown)  testing::TestSuite::Run
    @   00007FF6329C7C62  (unknown)  testing::internal::UnitTestImpl::RunAllTests
    @   00007FF6329D646C  (unknown)  testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl,bool>
    @   00007FF6329D6252  (unknown)  testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl,bool>
    @   00007FF6329C1B04  (unknown)  testing::UnitTest::Run
    @   00007FF6329A43F1  (unknown)  RUN_ALL_TESTS
    @   00007FF6329A1FA8  (unknown)  main
    @   00007FF632A0F839  (unknown)  __scrt_common_main_seh
    @   00007FFAD50884D4  (unknown)  BaseThreadInitThunk
    @   00007FFAD523E8B1  (unknown)  RtlUserThreadStart
]
[       OK ] ExtraTest.Basic (82 ms)
[ RUN      ] ExtraTest.MultiThreading
[       OK ] ExtraTest.MultiThreading (821 ms)
[----------] 2 tests from ExtraTest (903 ms total)

[----------] Global test environment tear-down
[==========] 2 tests from 1 test suite ran. (904 ms total)
[  PASSED  ] 2 tests.
```

This issue is the reason why the stack-trace feature was introduced except Windows (#24245)

**Comment by jtattermusch** [comment]
I'm pretty sure I know why on Bazel RBE windows (when you use the remote build) it doesn't work:
1. to get the stacktrace, you need the .pdb (which has debug symbols) alongside the .exe or dll.
2. bazel generates the PDB file correctly, but it's not copied to the RBE worker when tests are executed.
3. PDB is missing -> no stack trace on bazel RBE.

If you're getting the stack trace correctly with a local bazel build, but you're not getting one with remote build, this is almost certainly the reason.
Note that when using bazel RBE, AFAIK the current symbolizer (non-abseil) also doesn't work (but it works with local builds).

**Comment by jtattermusch** [comment]
The issue b/160675763 has an example how to work around the missing PDB file for selected targets but unfortunately it's a bit ugly solution so we likely cannot use it routinely for all our win tests.

**Comment by jtattermusch** [comment]
So if the windows symbolizer works for local bazel builds and it works with the workaround from b/160675763 (you can try it manually to be sure), we can probably go ahead with the win abseil symbolizer and get rid of our old (and a bit ugly) code from test_config.cc (e.g. https://github.com/grpc/grpc/pull/23420/files)?

**Comment by veblush** [comment]
Thanks, Jan for the context. It's surprising to know PDB is not supported on Bazel RBE yet, which puts it in an incompatible place. I think we should stay with a plain bazel on Window until it's addressed.

**Comment by jtattermusch** [comment]
while I agree that lack of debug symbols bazel RBE windows is a problem, I think now using Bazel RBE on windows would be a big step back (our tests are way too resource greedy to be able to test quickly on a single machine, especially if it's a developer's laptop). Bazel RBE is also paramount for being able to reproduce issues. (you need a controlled environment and ability to run many many iterations of our sometimes very slow tests).

**Comment by veblush** [comment]
Indeed. I agree with you that RBE is super helpful to fix the flaky issue.

How about having a plain bazel test as well on Windows? Only Windows doesn't seem to have it.

**Comment by jtattermusch** [comment]
I think it's fine to add a non-bazel windows bazel build-only job if you think that's useful (but so far I haven't seen a case where we were able to build with bazel RBE and not able to build with bazel locally on windows).
I don't think we should ad a local "bazel test" job as it would be too slow and we actually don't have "bazel test" jobs for C/C++ linux either (we only do "build-only" for linux on pull requests).
